XiEditor/Search.swift

```diff
@@ -200,6 +200,16 @@ extension EditViewController {
             (findViewController.searchField as? FindSearchField)?.resultCount = resultCount
         }
     }
+    
+    @IBAction func addNextToSelection(_ sender: AnyObject?) {
+        document.sendRpcAsync("selection_for_find", params: ["case_sensitive": false])
```

Comment:
not a problem unique to this PR (we're doing this in lots of places) but in the future I'd like us to stop reaching into the document and sending RPCs directly; the document (or something similar) should instead expose a function that handles sending the RPC. (no need to fix this now)